lisa/util/__init__.py

```diff
@@ -243,6 +243,14 @@ def __str__(self) -> str:
         return format_str
 
 
+class InternalTimeoutException(LisaException):
```

Comment:
`LisaTimeoutException`

Reply:
fixed

Reply:
Not seeing it's fixed...

Reply:
sorry, seems I missed this one, now it is fixed.